### PR TITLE
Remove the hash functions group

### PR DESCRIPTION
**Pull request by DelazJ**
Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

**Comment by DelazJ** [comment]
🙄 Because it does not(/no more?) exist in the GUI...

**Comment by havatv** [comment]
I see that now.
The group is gone, but the functions are there.  The descriptions of the functions should be moved to the "hash" function (and not removed).

From the description of the hash function in the field calculator:

> method | The hash method among 'md4', 'md5', 'sha1', 'sha224', 'sha384', 'sha512', 'sha3_224', 'sha3_256', 'sha3_384', 'sha3_512', 'keccak_224', 'keccak_256', 'keccak_384', 'keccak_512'

**Comment by havatv** [comment]
I still think it would be a shame to remove the existing documentation of the available hash methods. Could they not be moved somewhere else?

File: source/docs/user_manual/working_with_vector/expression.rst [hunk 1]
```diff
@@ -383,6 +383,10 @@ This group contains functions to convert one data type to another
    :header: "Function", "Description"
    :widths: auto
 
+   "hash", "Creates a hash (a string hex value) from a string with a given method
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/expression.rst]: I suggest that we add a section below this table  that lists the hash methods.

Hash methods
------------

.. csv-table::
   :header: "Function", "Description"
   :widths: auto

   "md4", "Message Digest algorithm. Produces a 128-bit (16-byte) hash value (widely used).

*and so on*

This (new section on the same level) has been used elsewhere in this part to add relevant information.

**Comment by DelazJ** [comment]
superseded by #5811

**Pull request closed without merging**